src/components/popover/popover.e2e.ts

```diff
@@ -734,4 +734,48 @@ describe("calcite-popover", () => {
         shadowFocusTargetSelector: `.${CSS.closeButton}`
       }));
   });
+
+  // refer to https://github.com/Esri/calcite-components/issues/5993 for context
```

Comment:
added this to provide context.

Reply:
Nice! WDYT about moving the comment inside the test, so if for whatever reason it's modified or moved, it goes with it?

Reply:
make sense.